examples/complete-dual-vpn-gateway/main.tf

```diff
@@ -0,0 +1,78 @@
+
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "vpn_gateway" {
+  source = "git::https://github.com/terraform-aws-modules/terraform-aws-vpn-gateway.git"
```

Comment:
I would set this to `source = "../.."`because that is easier to test. For instance while testing this PR I had to change this value to get your code.